Fixed spelling

TheOneWithTheBraid opened the pull request:
According to jugendhackt.org, the name is spelled `Jugend hackt`.

Commit:
Fixed spelling

marvinborner commented:
There are actually quite a bit more wrongly written "jugendhackt" phrases - I will merge them if you found everything!

marvinborner committed:
Fixed everything
Sorry for the force-push - I used sed for replacing and accidentally replaced the url etc
